release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,68 @@ If the following conditions are true, the feature is enabled by default:
+
+[id="ocp-4-15-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *Changed*
+** Modified the `NodeClockNotSynchronising` alerting rule to also consider the PTP protocol.
+
+[id="ocp-4-15-monitoring-new-metrics-server-to-access-metrics-API-technology-preview"]
+==== New Metrics Server component to access the Metrics API (Technology Preview)
+This release introduces a Technology Preview option to add a Metrics Server component to the in-cluster monitoring stack. 
+If enabled, Metrics Server, rather than Prometheus Adapter, collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
+Using Metrics Server instead of Prometheus Adapter frees the core platform Prometheus stack from handling this functionality.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-new-feature-to-send-exemplar-data-to-remote-write-storage-for-user-defined-projects"]
+==== New feature to send exemplar data to remote write storage for user-defined projects
+With this release, your user-defined projects can now use remote write to send exemplar data scraped by Prometheus to remote storage.
+To use this feature, configure remote write using the `sendExemplars` option in the `RemoteWriteSpec` resource.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#remotewritespec[RemoteWriteSpec] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-impoved-alert-querying-for-user-defined-projects"]
+==== Improved alert querying for user-defined projects
+With this release, applications in your user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+Specifically, you can now constuct queries that access the `/api/v1/alerts` endpoint via port 9093 for Thanos Querier, provided that the HTTP request contains a `namespace` parameter.
+In previous releases, the rules tenancy port for Thanos Querier did not provide API access to the `/api/v1/alerts` endpoint.
+
+[id="ocp-4-15-monitoring-prometheus-updated-to-tolerate-scrape-time-jitters"]
+==== Prometheus updated to tolerate scrape time jitters to reduce TSDB storage requirements
+With this release, the default Prometheus configuration in the in-cluster monitoring stack has been changed so that jitters are tolerated at scrape time, which improves chunk compression for data storage.
+As a result, the time series database uses significantly less disk space.
+Note that this change can reduce the accuracy of derived data, such as the running average of a time series.
```

Comment:
should we provide more details to make customers less worried? E.g. the loss of accuracy is barely noticeable.

cc @jan--f

Reply:
@simonpasquier and @jan--f  - I'd be fine with just omitting this sentence.

Reply:
works for me.